config/helm/eventing-auth-manager/templates/metrics-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
```

Comment:
Why do we need a service for EAM? Does some external components sends a REST call to EAM?

Reply:
This is just metrics service to expose metrics.

Reply:
Who will use this service?